dev-docs/modules/symitriDapRtdProvider.md

```diff
@@ -38,14 +38,16 @@ pbjs.setConfig({
     auctionDelay: 2000,
     dataProviders: [
       {
-        name: "dap",
+        name: "symitriDap",
```

Comment:
Was this wrong before or is this a breaking change?

Reply:
This was wrong before.